android-components/components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -948,6 +948,17 @@ sealed class TranslationsAction : BrowserAction() {
         val operation: TranslationOperation,
     ) : TranslationsAction(), ActionWithTab
 
+    /**
+     * Sets whether the device architecture supports translations or not.
+     *
+     * @property tabId The ID of the tab the [EngineSession] that requested the list.
+     * @property isEngineSupported If the engine supports translations on this device.
+     */
+    data class SetEngineSupportAction(
```

Comment:
```suggestion
    data class SetEngineSupportedAction(
```
Maybe? This is more about just using the same wording in the properties.

Reply:
Thanks, updated!